action.yml

```
@@ -12,6 +12,11 @@ inputs:
     description: "Your Wakatime API Key"
```

```suggestion
    description: "Your Wakatime/Wakapi/Hakatime API Key"
```
Maybe add more context to the API key variable since we're expanding it